### PR TITLE
Add a 4th node to Travis

### PR DESCRIPTION
**Pull request by maestromac**
## What type of PR is this? (check all applicable)
- [x] DX

## Description
- Explicitly separate jobs and cut down `script` size.
- Created a 4th node/job strictly for everything else that isn't RSpec.
- Remove Codeclimate test reporter since it hasn't been working.
## Related Tickets & Documents
n/a

## QA Instructions, Screenshots, Recordings
Please check out the build https://travis-ci.com/github/forem/forem/builds/214730360

## Added tests?
- [x] no, because they aren't needed

## Added to documentation?
- [x] no documentation needed

## [optional] Are there any post deployment tasks we need to perform?
n/a

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -97,22 +70,47 @@ notifications:
...
+      name: "Node 0"
+      env: KNAPSACK_PRO_CI_NODE_INDEX=0
+    - stage: Test
+      name: "Node 1"
+      env: KNAPSACK_PRO_CI_NODE_INDEX=1
+    - stage: Test
+      name: "Node 2"
+      env: KNAPSACK_PRO_CI_NODE_INDEX=2
+    - stage: Test
+      name: "Preact, Storybook, bundle audit, console check, e2e"
```
Comment (nickytonline) [inline .travis.yml:83]: nitpick (non-blocking): You don't need to mention Preact. Maybe instead put front-end tests.

**Review by nickytonline — APPROVED** [review]
LGTM. Left a non-blocking comment. 🚀

File: .travis.yml (hunk 1 above)
Comment (maestromac) [reply .travis.yml:83]:
```suggestion
      name: "Front-end tests, Storybook, bundle audit, console check, e2e"
```
:+1: lets change it.

File: .travis.yml [hunk 2]
```diff
@@ -57,34 +50,14 @@ script:
   - echo -e '-XX:+DisableExplicitGC\n-Djdk.io.permissionsUseCanonicalPath=true\n-Dlog4j.skipJansi=true\n-server\n' | sudo tee -a /etc/elasticsearch/jvm.options
   - sudo chown -R elasticsearch:elasticsearch /etc/default/elasticsearch
   - sudo systemctl start elasticsearch
-  - curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
-  - chmod +x ./cc-test-reporter
```
Comment (rhymes) [inline .travis.yml:61]: bye bye miss american pie

**Review by rhymes — APPROVED** [review]
Cool!

**Comment by mstruve** [comment]
Excited to see what this does to our build times!

**Pull request merged**